### PR TITLE
#206 Molekul - FMSearch

### PR DESCRIPTION
**Pull request by bryantobing12**
### Detailed description of changes

- Outlined Input border radius set to 8px
- Search icon adornment to use primary color
- Storybook example

### Recommendations on how to test the changes

- Check if the component match the design

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/206

#### Before

image

#### After

image

### CODE REVIEW

File: src/Components/Molecules/FMSearch/index.tsx [hunk 1]
```diff
@@ -25,11 +26,10 @@ export const FMSearch = memo((props: IFMSearch) => {
...
       value={value}
-      placeholder="Search"
       InputProps={{
         startAdornment: (
           <InputAdornment position="start">
-            <SearchIcon color="secondary"></SearchIcon>
+            <SearchIcon color="primary"></SearchIcon>
           </InputAdornment>
         ),
       }}
```
Comment (muhammad-farhan64) [inline src/Components/Molecules/FMSearch/index.tsx:35]: ini mungkin bisa di ganti dengan adorment dan adormentPosiition propsnya

Comment (bryantobing12) [reply src/Components/Molecules/FMSearch/index.tsx]: updated @muhammad-farhan64

File: src/Components/Molecules/FMSearch/FMSearch.interfaces.tsx [hunk 2]
```diff
@@ -1,14 +1,7 @@
...
   testID: `search-${string}`;
-  label?: string;
-  adornment?: ReactChild;
-  helperText?: string;
-  onChange?: any;
-  value?: any;
-  variants?: "standard" | "outlined" | "filled";
-  type?: string;
-  disabled?: boolean;
+  sx?: StandardTextFieldProps["sx"];
```
Comment (gilangdarmawan) [inline src/Components/Molecules/FMSearch/FMSearch.interfaces.tsx:6]: ada concern kenapa add `sx`? soalnya di component nya sx ga dipake

File: src/Components/Molecules/FMSearch/index.tsx [hunk 3]
```diff
@@ -1,39 +1,26 @@
...
-      placeholder="Search"
-      InputProps={{
-        startAdornment: (
+    <div data-testid={testID}>
+      <FATextField
+        testID="input-search"
+        adornment={
           <InputAdornment position="start">
-            <SearchIcon color="secondary"></SearchIcon>
+            <SearchIcon color="primary"></SearchIcon>
```
Comment (gilangdarmawan) [inline src/Components/Molecules/FMSearch/index.tsx:17]: Bisa pakai `FAIcon` sebagai alternatif

File: src/Components/Molecules/FMSearch/index.tsx [hunk 4]
```diff
@@ -1,39 +1,26 @@
...
-      disabled={disabled}
-      type={type}
-      label={label}
-      variant={variants}
-      onChange={onChange}
-      value={value}
-      placeholder="Search"
-      InputProps={{
-        startAdornment: (
+    <div data-testid={testID}>
```
Comment (gilangdarmawan) [inline src/Components/Molecules/FMSearch/index.tsx:12]: Boleh diganti ke `<Box>`

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

File: src/Components/Molecules/FMSearch/FMSearch.interfaces.tsx (hunk 2 above)
Comment (bryantobing12) [reply src/Components/Molecules/FMSearch/FMSearch.interfaces.tsx:6]: di `FOHeader` FMSearch nya make `sx` . tapi ini juga yang seharusnya ga boleh sih karena dicustom inline gitu. 
image

**Review by gilangdarmawan — APPROVED** [review]

**Pull request merged**